README.md

```diff
@@ -27,11 +27,48 @@ TODO: Write documentation for how to build initrd locally
 
 See [Hacking](HACKING.md)
 
-# Integrating with Kernel Snap
+# Releasing
 
-When your initrd DEB package is built, you need to perform following steps to make it a part of kernel snap:
+The UC initrd is included in kernel snaps. However, the process to get
+it inside the kernel snap is not immediate and consists of a few
+steps. First, we need to build the `ubuntu-core-initramfs`
+debian package in the [snappy-dev/image
+PPA](https://launchpad.net/~snappy-dev/+archive/ubuntu/image) by
+following these steps:
 
-TODO: Write documentation for how to integrate initrd.img to your (custom or canonical) kernel 
+1. Update the changelog with latest changes since last release (use `dch -i` for this)
```

Comment:
```suggestion
1. Update the changelog with latest changes since last release (use `dch -i` for this)
1. Commit using `debcommit --release -a` which will commit the changelog update & tag the release
```